pages/glossary/inspection-states.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Inspection states
+key: inspection-states
+unambiguous: true
+objective: true
+---
+
+The possible _inspection states_ of an HTML element are
+
+- the element is [focused][] and [hovered][]; or
+- the element is [focused][] but not [hovered][]; or
+- the element is not [focused][] but [hovered][]; or
+- the element is neither [focused][] nor [hovered][].
```

Comment:
Because we defined "inspection states" to be the combination of all focused and hovered states :-)
And the neither focused nor hovered is one of the adequate inspection states that is checked in the expectation, i.e., if the link has a different style when it is not focused neither hovered then it meets the expectation.